Jasmine/Common/ViewModels/TimedViewModelProtocol.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+
+/// A ViewModel that implements a timer, i.e. it is timed.
+protocol TimedViewModelProtocol: BaseViewModelProtocol {
```

Comment:
Not really sure about the strong/weak, I think we can just try weak haha. This is to prevent retain cycle?

Hmm yeah timer should only have VM right. But then how do I get the VC from the VM? The base VM does not have VC.

If I make VM have BaseVC, there is a problem with the GridVM/TetrisVM protocols since the VC must be specific GridVC/TetrisVC.